provision/openshift/logging.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
The ElasticSearch operator failed to start on a fresh installation on gh-keycloak with the message: 

> failed to populate resolver cache from source operatorgroup-unavailable/openshift-operators-redhat: found 0 operatorgroups in namespace openshift-operators-redhat: expected 1

In a previous setup, if you installed it via the UI first, that group might have already existed. See PR https://github.com/keycloak/keycloak-benchmark/pull/415 for such an operator group (usually they should automatically named, I picked a static name for this instance)